plugin/src/openshift/utils/KialiIntegration.ts

```diff
@@ -26,6 +27,14 @@ export const getPluginConfig = async function (): Promise<PluginConfig> {
   });
 };
 
+// Set the router basename where OSSMC page is loaded
+export const setRouterBasename = (pathname: string): void => {
+  const ossmConsoleIndex = pathname.indexOf('/ossmconsole');
```

Comment:
I think that this plugin base name could be better defined as a constant, so it could be identified easily in case of change or used in other places.

Reply:
do you mean define `/ossmconsole` as a constant?

Reply:
Yes, right